### PR TITLE
Update capacity-planning-for-active-directory-domain-services.md

### PR DESCRIPTION
**Pull request by kenbrumf**
Clarified storage performance considerations for more complex scenarios.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@kenbrumf : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@kenbrumf : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Teresa-Motiv — APPROVED** [review]

**Comment by Teresa-Motiv** [comment]
#sign-off

File: WindowsServerDocs/administration/performance-tuning/role/active-directory-server/capacity-planning-for-active-directory-domain-services.md [hunk 1]
```diff
@@ -290,13 +290,14 @@ In a scenario where multiple Virtual Hard Disk (VHD) files are being allocated o
...
...larification, these are not “bad” configurations, they are more complex configurations that require every component along the way to be working properly, thus requiring additional attention to ensuring that performance is acceptable.  See Appendix C, subsection “Introducing SANs” and Appendix D later in this document for more detailed explanations.  Also, whereas Solid State Drives do not have the limitation of spinning disks (Hard Drives) regarding only allowing 1 IO at a time to be processed, they do still have IO limitations and overloading/oversubscribing of SSDs is possible.  In short, the end goal of all storage performance efforts, regardless of underlying storage architecture and design, is to ensure the needed amount of Input/Output Operations per Second (IOPS) are available and that those IOPS happen within an acceptable time frames (as specified elsewhere in this document).  For those scenarios with locally attached storage, reference the Appendix C for the basics in how to design traditional local storage scenarios.  These principals are generally applicable to more complex storage tiers and will also help in dialog with the vendors supporting backend storage solutions.
```
Comment (illfated) [inline WindowsServerDocs/administration/performance-tuning/role/active-directory-server/capacity-planning-for-active-directory-domain-services.md:300]:
```suggestion
Specifically, shared storage (RAID, SAN, NAS, JBOD (i.e. Storage Spaces), VHD) all have the ability to be oversubscribed/overloaded by other work loads that are placed on the back end storage.  They also add in the challenge that SAN/network/driver issues (everything between the physical disk and the AD application) can cause throttling and/or delays. For clarification, these are not “bad” configurations, they are more complex configurations that require every component along the way to be working properly, thus requiring additional attention to ensure that performance is acceptable.  See Appendix C, subsection "Introducing SANs", and Appendix D later in this document for more detailed explanations. Also, whereas Solid State Drives do not have the limitation of spinning disks (Hard Drives) regarding only allowing 1 IO at a time to be processed, they do still have IO limitations, and overloading/oversubscribing of SSDs is possible.  In short, the end goal of all storage performance efforts, regardless of underlying storage architecture and design, is to ensure the needed amount of Input/output Operations Per Second (IOPS) are available and that those IOPS happen within an acceptable time frame (as specified elsewhere in this document). For those scenarios with locally attached storage, reference the Appendix C for the basics in how to design traditional local storage scenarios. These principals are generally applicable to more complex storage tiers and will also help in dialog with the vendors supporting backend storage solutions.
```
Some adjustments for grammar, punctuation, and readability.

**Comment by PRMerger6** [comment]
@kenbrumf : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Teresa-Motiv — CHANGES_REQUESTED** [review]
subsection "Introducing SANs",
   The comma should be inside the quote marks.

allowing 1 IO at a time
    Spell out "one"

**Comment by PRMerger6** [comment]
@kenbrumf : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/performance-tuning/role/active-directory-server/capacity-planning-for-active-directory-domain-services.md [hunk 2]
```diff
@@ -290,13 +290,14 @@ In a scenario where multiple Virtual Hard Disk (VHD) files are being allocated o
...
... clarification, these are not “bad” configurations, they are more complex configurations that require every component along the way to be working properly, thus requiring additional attention to ensure that performance is acceptable.  See Appendix C, subsection "Introducing SANs," and Appendix D later in this document for more detailed explanations. Also, whereas Solid State Drives do not have the limitation of spinning disks (Hard Drives) regarding only allowing one IO at a time to be processed, they do still have IO limitations, and overloading/oversubscribing of SSDs is possible.  In short, the end goal of all storage performance efforts, regardless of underlying storage architecture and design, is to ensure the needed amount of Input/output Operations Per Second (IOPS) are available and that those IOPS happen within an acceptable time frame (as specified elsewhere in this document). For those scenarios with locally attached storage, reference the Appendix C for the basics in how to design traditional local storage scenarios. These principals are generally applicable to more complex storage tiers and will also help in dialog with the vendors supporting backend storage solutions.
```
Comment (illfated) [inline WindowsServerDocs/administration/performance-tuning/role/active-directory-server/capacity-planning-for-active-directory-domain-services.md:300]: I missed out on one extra set of quotation marks to change from curly quotes to straight quotes (these are not “bad” configurations):
```suggestion
Specifically, shared storage (RAID, SAN, NAS, JBOD (i.e. Storage Spaces), VHD) all have the ability to be oversubscribed/overloaded by other work loads that are placed on the back end storage. They also add in the challenge that SAN/network/driver issues (everything between the physical disk and the AD application) can cause throttling and/or delays. For clarification, these are not "bad" configurations, they are more complex configurations that require every component along the way to be working properly, thus requiring additional attention to ensure that performance is acceptable.  See Appendix C, subsection "Introducing SANs," and Appendix D later in this document for more detailed explanations. Also, whereas Solid State Drives do not have the limitation of spinning disks (Hard Drives) regarding only allowing one IO at a time to be processed, they do still have IO limitations, and overloading/oversubscribing of SSDs is possible. In short, the end goal of all storage performance efforts, regardless of underlying storage architecture and design, is to ensure that the needed amount of Input/output Operations Per Second (IOPS) is available and that those IOPS happen within an acceptable time frame (as specified elsewhere in this document). For those scenarios with locally attached storage, reference Appendix C for the basics in how to design traditional local storage scenarios. These principals are generally applicable to more complex storage tiers and will also help in dialog with the vendors supporting backend storage solutions.
```
Simplified a couple of extra double spaces between sentences. Simplified "reference the Appendix C" by removing the definite article "the", in compliance with its most common expression usage. Also: "to ensure that the needed amount" — "is available" (amount/singular).

**Comment by PRMerger15** [comment]
@kenbrumf : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me, I am happy with this result.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@illfated: I'm sorry - only the author of this article, @teresa-motiv, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is based on an agreement with the MS Docs team.

**Comment by Teresa-Motiv** [comment]
#Sign-off

**Review by Teresa-Motiv — APPROVED** [review]

**Comment by Teresa-Motiv** [comment]
#label:"sign-off"

**Comment by Teresa-Motiv** [comment]
@JasonGerend Can you merge this PR, or do you know who can?

Thanks

**Comment by Teresa-Motiv** [comment]
#sign-off

**Comment by illfated** [comment]
@denisebmsft : Would you be able to merge this PR? I mean, it has been verified by the assigned author, who don't have merging rights, though I don't know who is the correct person to perform the merge.

**Pull request merged**